packages/plain-object/README.md

```
@@ -196,57 +196,68 @@ someFn(value);
+    1.27x faster than "zod":"4.4.3": 'zod.util.isPlainObject(v)'
+    2.17x faster than "is-plain-object":"5.0.0": 'isPlainObject(v)'
+    2.65x faster than "immer/is-plain-object":"4.2.0": 'isPlainObject(v)'
+    3.12x faster than "redux":"5.0.1": 'isPlainObject(v)'
+    75.67x faster than lodash-es:"4.18.1": '_.isPlainObject(v)'
+
 ```
 
-### Bun 1.3.12
+### Bun 1.3.13
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)